src/Components/NewTokenButton.jsx

```diff
@@ -14,8 +14,8 @@ export default function Button({ setToken }) {
   };
 
   return (
-    <div>
-      <button onClick={handleClick}>Get new token</button>
+    <div className="token">
```

Comment:
I like how you're gradually repurposing and refactoring components to suit the app!